app/api/routes.py

```diff
@@ -94,13 +94,15 @@ def add_to_garden():
 
 
     result = {
+    'garden_plant_id': garden_plant.id,
```

Comment:
This wording is a little weird, but it does differentiate from plants and gardens and such. It might be nice if we can figure out a better way to test this so we can only return a little information so we don't have to have such a crazy name!